__tests__/app.test.js

```diff
@@ -114,7 +114,8 @@ describe('app', () => {
               body: 'some gifs',
               created_at: '2020-11-03T08:12:00.000Z',
               votes: 0,
-              article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+              article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
```

Comment:
We shouldn't ever go back and change old tests. The additional functionality of a comment count should be tested in it's own new test which only checks for the comment_count property and the existing tests should be robust enough to still pass. Please take car not to go back and change tests once they have passed